Sources/HaCWebsiteLib/Models/Workshop+init.swift

```diff
@@ -53,6 +52,7 @@ extension Workshop {
   }
 
   private static func getMetadata(localPath: String) throws -> Yaml {
+    print (localPath + filePaths.metadata)
```

Comment:
Why is this being printed? If the goal is to log it, why not use our logging system?

Reply:
Was just there for debugging. Good catch